doc/source/getting-started/usage.rst

```diff
@@ -1,30 +1,31 @@
 Usage instructions
 ==================
 
-After the installation, in ``<qibosoq-folder>/src/qibosoq`` you will find the ``__main__.py`` file.
-Here, the IP and port of the server are hardcoded and modifiable (if you installed qibosoq in developer mode).
+After the installation, in ``<qibosoq-folder>/src/qibosoq`` you will find the ``configuration.py`` file.
+Here, various qibosoq parameter are hardcoded and modifiable (if you installed qibosoq in developer mode).
```

Comment:
We can also treat them as default parameters, and modify with environment variables (such that you don't have to modify the source).

Reply:
```sh
❯ export CIAO=3
❯ sudo echo $CIAO
3
❯ python3 -c "import os; print(os.environ['CIAO'])"
3
❯ sudo python3 -c "import os; print(os.environ['CIAO'])"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "<frozen os>", line 679, in __getitem__
KeyError: 'CIAO'
❯ sudo -E python3 -c "import os; print(os.environ['CIAO'])"
3
```

Reply:
nice, thanks!